timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -706,7 +719,10 @@ def fit(
             )
 
         if tuning_data is not None:
-            tuning_data = self._check_and_prepare_data_frame(tuning_data, name="tuning_data")
+            # TODO: Allow passing separate static_features to tuning_data?
```

Comment:
this is problematic imo. did we have anything constraining tuning data to be the same set of items as the training data before? if not, this implicitly does this and is a breaking change.

Reply:
All right, let's postpone this discussion to a later point. I will move the docstring updates to a separate PR.